packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -0,0 +1,259 @@
+import React from 'react';
+import { Button, Dialog, Icon, Loading, Popover, Table, Tabs } from '@deriv/components';
+import { generateHexColourFromNickname, getShortNickname } from 'Utils/string';
+import { height_constants } from 'Utils/height_constants';
+import Dp2pContext from 'Components/context/dp2p-context';
+import { InfiniteLoaderList } from 'Components/table/infinite-loader-list.jsx';
+import { requestWS } from 'Utils/websocket';
+import Popup from '../buy-sell/popup.jsx';
+import { buy_sell } from '../../constants/buy-sell';
+import { localize } from '../i18next';
+import './advertiser-page.scss';
+
+const RowComponent = React.memo(({ data, is_buy_advert, showAdPopup, style }) => {
+    const { advertiser_id } = React.useContext(Dp2pContext);
+    const is_my_ad = data.advertiser_id === advertiser_id;
+
+    return (
+        <div style={style}>
+            <Table.Row className='advertiser-page__adverts-table_row'>
+                <Table.Cell>
+                    {data.display_min_available}&ndash;{data.display_max_available} {data.offer_currency}
+                </Table.Cell>
+                <Table.Cell className='advertiser-page__adverts-price' flex='2fr'>
```

Comment:
i dont think we have flex properties in Table.Cell?

Reply:
You're right, I followed the same pattern as buy sell table, this is removed now. Also removed from `row.jsx`.